docs/api/rpc-params.md

```diff
@@ -0,0 +1,158 @@
+<strong>heads up</strong><br><br>
+
+This page refers to a **technical preview**.
+
+These features are not currently available on TestNet.  They will be available in an upcoming release.
+
+</blockquote>
+
+This is an improvement of our RPC API that allows querying account info, access key(s), and state by a given block identifier (either block height or block hash).
+
+## Introduction
+
+The following identifiers are used throughout the code samples below. 
+
+- `account_id`: (string) NEAR [account ID format](/docs/concepts/account)
+- `prefix`: (base64 string) optional string used to segment our key-value store
+- `public_key` (string) the public part of your public/private key pair
+- `method_name`: (string) the name of the RPC method you're calling
+
+## Common Parameters
+
+- `block_id`: (integer) block height or block hash
+- `finality`: (string) filter for status of block finalization [ `optmistic` | `near-final` | `final` ] 
+
+
+*At time of writing, `finality` is only available on StageNet.*
+
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+
+### Using block height
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using block hash
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `optimistic` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `near-final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "near-final"
+             }'
+```
+
+### Using `final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "final"
+             }'
+```
+
+<blockquote class="info">
+<strong>did you know?</strong><br><br>
```

Comment:
Since it is the only place we document this, I guess it is quite sarcastic to expect people to know this :smile:

I think we want to document this in the interaction/rpc document, and then this "did you know?" section will make sense, and it can be short with just a few words and a link to the section in the interaction/rpc document.

Reply:
i think this is an English thing -- in the US, it's quite natural to say to someone, "hey, did you know [random fact X about Y]?" and there's no sarcasm at all.  it's like news or an update or something.  here in our docs this is a consistent blue callout in many places that brings attention to something that would be useful to know

Reply:
@amgando TIL. Thanks!

I think there is still a subtle difference in the case when this is *the only* place you could learn about the fact. Though, I trust your judgement.